<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QPushButton" name="lUsuario">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>usuario</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lClave">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>contrase√±a</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUsuario">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>261</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leClave">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>261</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbIniciar">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>iniciar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leURL">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>361</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
